02-esc/modal.js

```diff
@@ -35,29 +39,30 @@ function openModal() { //(e)
 
 	let openModalStepTwoBtn = document.querySelector(".open-modal-step-2");
 	openModalStepTwoBtn.addEventListener("click", checkModalStepOneInput);
+}
 
-	function checkModalStepOneInput(e) {
-		e.preventDefault();
-		let ok = false;
-        let dateInput = document.querySelector("#date");
+function checkModalStepOneInput(e) {
+    e.preventDefault();
+    let ok = false;
+    let dateInput = document.querySelector("#date");
+    
+    if (dateInput.value == "") {
+        dateInput.style.outline = 'solid 2px red';
+        dateInput.addEventListener("click", normalBorderColor);
         
-        if (dateInput.value == "") {
-            dateInput.style.border = 'solid 3px red';
-            dateInput.addEventListener("click", normalBorderColor);
-            
-            function normalBorderColor(){ 
-                dateInput.style.border = 'inset 2px rgb(133, 133, 133)';
-            }
-
-        } if (dateInput.value !== "") {
-			ok = true;
-		} if (ok) {
-			getDateFromForm(e);
-			openModalStepTwo(e);
-		}
-	}
+        function normalBorderColor(){ 
+            dateInput.style.outline = 'none';
+        }
+
+    } if (dateInput.value !== "") {
+        ok = true;
+    } if (ok) {
```

Comment:
These if statements are formatted as if they were `else if` statements, but they're not. This will confuse readers of the code.

On top of this, the last two if statements could have been written as a single one. The second statement sets the variable `ok` to `true` and the third checks that `ok` is `true`. That means that it is always the case that if the second is true, the third is true, and if the second is false, the third is false.

Finally, the condition on line 57 is the exact opposite of the condition on line 49, so it could just be an `else` instead.

I would replace these three lines with:

```
} else {
```